layouts/partials/social-icons.html

```diff
@@ -6,7 +6,7 @@
     {{ $icon := anchorize . }} 
     {{ if isset $currentPage.Site.Params $icon }}
       <a class="social-icons__icon" title="{{ . }}"
-         style="background-image: url(/images/social/{{ $icon }}.svg)" 
+         style="background-image: url('{{print "images/social/" $icon ".svg" | absURL}}')" 
```

Comment:
Looks good to me, although the themes directory shows an example with a different structure:

```
url('{{ "images/background.jpg" | absURL }}')
```

But I'm thinking `print` is used because we have a variable involved.

Reply:
yup correct, we need `print` to concatenate strings